modules/installation-configuration-parameters.adoc

```diff
@@ -1159,6 +1203,31 @@ The GCP Compute Engine System service account email, like `<service_account_name
 |For compute machines, the ID of the project in which the KMS key ring exists. This value defaults to the VM project ID if not set.
 |The GCP project ID.
 
+|`compute.platform.gcp.osDisk.encryptionKey.kmsKeyServiceAccount`
+|The GCP service account used for the encryption request for compute machines. If absent, the Compute Engine default service account is used. For more information on GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
```

Comment:
If the account is absent? If the value is blank? If the property is not included in the YAML?

Reply:
Reworded as "If this value is not set,"